src/main/java/net/minecraftforge/fml/relauncher/CoreModManager.java

```diff
@@ -64,7 +64,8 @@
     private static final Attributes.Name MODTYPE = new Attributes.Name("ModType");
     private static final Attributes.Name MODSIDE = new Attributes.Name("ModSide");
     private static final Attributes.Name MODCONTAINSDEPS = new Attributes.Name("ContainedDeps");
-    private static String[] rootPlugins = { "net.minecraftforge.fml.relauncher.FMLCorePlugin", "net.minecraftforge.classloading.FMLForgePlugin" };
+    private static String[] rootPlugins = { "net.minecraftforge.fml.relauncher.FMLCorePlugin", "net.minecraftforge.classloading.FMLForgePlugin",
```

Comment:
Not how you add your mod in. You can just add it to the list of mods that are always loaded, in FML, or use the "forge hook" spot.